src/System.Runtime.CompilerServices.Unsafe/Directory.Build.props

```diff
@@ -1,6 +1,7 @@
 ﻿<Project>
   <Import Project="..\Directory.Build.props" />
   <PropertyGroup>
+  	<PackageVersion>4.7.1</PackageVersion>
     <AssemblyVersion>4.0.6.0</AssemblyVersion>
```

Comment:
shoudnt we update the assembly version as well ?

Reply:
We explicitly don't want to do this in order to not cause additional binding redirect issues for consumers. We thought about doing this which would help with the case of people adding this version to the GAC, but we decided that if they did, they most likely also GACed the rest of the netstandard facades in which case they wouldn't require this fix. cc: @ericstj